networkx/algorithms/approximation/steinertree.py

```diff
@@ -0,0 +1,76 @@
+import networkx as nx
+from itertools import combinations, chain
+from networkx.utils import pairwise, not_implemented_for
+
+__all__ = ['metric_closure', 'steiner_tree']
+
+
+@not_implemented_for('directed')
+def metric_closure(G, nodes, weight="weight"):
+    """  Returns the sub-graph of the metric closure of the graph G induced by
+        the nodes.
+
+    Parameters:
+    -----------
+    G : Networkx graph
+
+    nodes : List
+            The function returns the sub-graph of metric closure induced by the
+            nodes specified in this list.
+
+    Returns:
```

Comment:
Please check the format for the `Returns` section, as described in the [numpydoc][1] specification.

[1]: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections